Resources/Prototypes/Entities/Items/Clothing/OuterClothing.yml

```diff
@@ -781,7 +781,7 @@
 - type: entity
   parent: OuterclothingBase
   id: OuterclothingIhsrig
-  name: IHS rig
+  name: ihs rig
```

Comment:
Looks like (?) an acronym (also occurs below).

Reply:
Since this is taken from Eris it's probably IronHammer Security.